core/modules/system/system.install

```diff
@@ -538,6 +538,19 @@ function system_requirements($phase) {
       );
     }
   }
+  
+  if ($phase == 'runtime') {
+    // Check if display error messages enabled
+    $error_level = config_get('system.core', 'error_level');
+    if ($error_level != ERROR_REPORTING_HIDE) {
+      $requirements['error_level'] = array(
+        'title' => $t('Display error messages'),
+        'value' => $t('Enabled'),
+        'severity' => REQUIREMENT_WARNING,
+        'description' => $t('Backdrop CMS provides the ability to display error messages, which may be useful on development environment. The site running on production environment should not display any errors. Do not forget to <a href="@error_level">turn off display errors</a> for production environment.', array('@error_level' => base_path() . 'admin/config/development/logging')),
```

Comment:
Recommended text: 
`Backdrop CMS provides the ability to display error messages, which can be useful while a site is in development. Do not forget to <a href="@error_level">disable the display of errors</a> when this site is in use.'

Recommended changes ,and thoughts behind them:
* less verbose
* removed the word `environment` since some sites only have one.
* tried to find a more friendly way to say "production"